tests/test_preload_script.py

```diff
@@ -30,9 +31,17 @@ async def test_addPreloadScript_setGlobalVariable(websocket, context_id):
             }
         })
 
-    resp = await read_JSON_message(websocket)
+    assert result == {'script': ANY_STR}
 
-    assert resp == {'id': 1, 'script': '1'}
+    await execute_command(
```

Comment:
The new script should be run only on the new frames. So navigation is needed.

Reply:
Ack. Resolved.